# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-font-face.c
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/cairo-font-face.c
--- a/gfx/cairo/cairo/src/cairo-font-face.c	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-font-face.c	Mon May 06 19:25:12 2024 +0000
@@ -133,7 +133,6 @@
 
     return font_face;
 }
-slim_hidden_def (cairo_font_face_reference);
 
 static inline cairo_bool_t
 __put(cairo_reference_count_t *v)
@@ -190,7 +189,6 @@
 
     free (font_face);
 }
-slim_hidden_def (cairo_font_face_destroy);
 
 /**
  * cairo_font_face_get_type:
@@ -272,7 +270,6 @@
     return _cairo_user_data_array_get_data (&font_face->user_data,
 					    key);
 }
-slim_hidden_def (cairo_font_face_get_user_data);
 
 /**
  * cairo_font_face_set_user_data:
@@ -304,7 +301,6 @@
     return _cairo_user_data_array_set_data (&font_face->user_data,
 					    key, user_data, destroy);
 }
-slim_hidden_def (cairo_font_face_set_user_data);
 
 void
 _cairo_unscaled_font_init (cairo_unscaled_font_t               *unscaled_font,